Radzen.Blazor/DropDownBase.cs

```diff
@@ -598,7 +598,7 @@ private async System.Threading.Tasks.Task HandleKeyPress(Microsoft.AspNetCore.Co
                     //
                 }
             }
-            else if (Multiple && key == "Space")
```

Comment:
This is a breaking change and we can't accept it. Enter closes the dropdown at the moment and this is why space selects and item.

Reply:
Hi @akorchev !

Is there any possibility we could make this change through?

## Using `Space` is not convenient and buggy.

Consider the following scenario:

Your dropdown contains a list of names (first name + last name):

- Dmitry Vychikov
- Dmitry Tikhomirov
- Dmitry ...
- Dmitry ... (a bunch of different Dmitry's)

Then, 
- You start typing `Dmitry`, then see a lot of them
- You move the cursor with arrow buttons to find what you need, but still there are too many of them
- You decide to add a surname to the search, so you start typing again
- You press `<Space>`

But instead of putting a space character in the input, it selects an item! This is very frustrating and not user friendly.

## UI/UX

> Enter closes the dropdown at the moment and this is why space selects and item.

I would say, most users expect that `Esc` button closes the dropdown, not `Enter`.


## Possible options or next steps

- @akorchev are u considering to improve this at all?
- Would you be happy if we made this improvement under say a feature flag or as a configurable parameter? We could leave the default settings in place for now, but provide consumers with the ability to override this behaviour.
    - So it wouldn't be a breaking change anymore